### ISSUE TITLE
Try out TA's Suggestions to improving inference.

### ISSUE THREAD

**Issue by amuamushu** [amuamushu/adv_avod_ssn#23]
Talked to a TA about improving runtime of existing code. His suggestions are

1. make sure the model is not updating the weights --> pytorch: model.train() is off
2. input in the inference doesn't need gradient computations --> pytorch: input.requires_grad(False)
3. other direction: increase batch_size as much as memory fits

Make sure `nvidia-smi` shows percentage when it's running.

_Originally posted by @amuamushu in https://github.com/amuamushu/adv_avod_ssn/issues/6#issuecomment-1029469546_

**Comment by amuamushu** [comment]
won't do bc of time constraint
